src/LoginDialog/OidcLoginButtons.tsx

```diff
@@ -11,6 +11,29 @@ import { generateOidcLoginRedirectUrl } from './OidcUtilities';
 
 import './OidcLoginButtons.scss';
 
+const knownProfilePrefixes: { [key: string]: string } = {
+  'https://auth0.auth0.com': 'auth0',
+  'https://login.microsoftonline.com': 'microsoft',
+  'https://elasticpath.okta.com': 'okta',
+  'https://accounts.google.com': 'google',
+  'https://appleid.apple.com': 'apple',
+  'https://login.salesforce.com': 'salesforce',
```

Comment:
We need to provide our customers documentation on how to add/remove providers from this list. Additionally how to configure these through the dashboard.

Reply:
Any reason why we're including these 6 OOTB as oppose to others? Are these the most-used that we want to make available in the storefront?

Reply:
Looking further through this PR, this could be more dynamic so the knownProfilePrefixes are retrieved from config.ts and the prefix is used for the button styling. @Danial-EP @epmkelly @dusanradovanovic thoughts on the above?

Reply:
the prefix is something that matches specifically with the styles, if we wanted it to be more dynamic with the prefixes, we'd need to do similar for the styling as well. These button styles are something that are really are choosing, but styles from the providers themselves. I don't think it's worth making it more dynamic in my opinion. 

Not sure on why these 6, but they seem most-used in my opinion. 

definitely we should have some docs on how to remove/add, this should also show where/how to add the styles for the button as well.